DiscordManimator.py

```diff
@@ -1,11 +1,14 @@
+import asyncio
 import discord
 import docker
 import os
 import tempfile
+import re
 
 from discord.ext import commands
 from dotenv import load_dotenv
 from pathlib import Path
+from io import BytesIO
```

Comment:
Would prefer for just `io` to be imported and then this accessed through `io.BytesIO`

Reply:
Any particular reason why? To me it seems cleaner to import just what we need (though I guess you could make that argument for `os` and `tempfile` as well, IDK).

Reply:
It makes it more explicit where `BytesIO` is coming from, which helps readability. And afaik the whole module gets run anyway so you're not like, only running the stuff that `BytesIO` needs, so I don't see any real benefit to how it currently is